modules/reindex/src/internalClusterTest/java/org/elasticsearch/migration/SystemIndexMigrationIT.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.migration;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.elasticsearch.action.admin.cluster.migration.PostFeatureUpgradeAction;
+import org.elasticsearch.action.admin.cluster.migration.PostFeatureUpgradeRequest;
+import org.elasticsearch.cluster.ClusterStateListener;
+import org.elasticsearch.cluster.service.ClusterService;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.persistent.PersistentTasksCustomMetadata;
+import org.elasticsearch.plugins.Plugin;
+import org.elasticsearch.reindex.ReindexPlugin;
+import org.elasticsearch.test.ESIntegTestCase;
+import org.elasticsearch.test.InternalTestCluster;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.BrokenBarrierException;
+import java.util.concurrent.CyclicBarrier;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import static org.elasticsearch.upgrades.SystemIndexMigrationTaskParams.SYSTEM_INDEX_UPGRADE_TASK_NAME;
+
+/**
+ * This class is for testing that when restarting a node, SystemIndexMigrationTaskState can be read.
+ */
+@ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.TEST, numDataNodes = 0, numClientNodes = 0, autoManageMasterNodes = false)
+public class SystemIndexMigrationIT extends AbstractFeatureMigrationIntegTest {
+    private static Logger logger = LogManager.getLogger(SystemIndexMigrationIT.class);
+
+    @Override
+    protected Settings nodeSettings(int nodeOrdinal, Settings otherSettings) {
+        return Settings.builder().put(super.nodeSettings(nodeOrdinal, otherSettings)).build();
+    }
+
+    @Override
+    protected boolean forbidPrivateIndexSettings() {
+        // We need to be able to set the index creation version manually.
+        return false;
+    }
+
+    @Override
+    protected Collection<Class<? extends Plugin>> nodePlugins() {
+        List<Class<? extends Plugin>> plugins = new ArrayList<>(super.nodePlugins());
+        plugins.add(TestPlugin.class);
+        plugins.add(ReindexPlugin.class);
+        return plugins;
+    }
+
+    public void testSystemIndexMigrationCanBeInterruptedWithShutdown() throws Exception {
```

Comment:
for the record. The logs from the test. Even though it passes, I wanted to double check this is fine
```
/Library/Java/JavaVirtualMachines/jdk-17.0.2.jdk/Contents/Home/bin/java -ea -Djava.locale.providers=SPI,COMPAT --add-opens=java.base/java.security.cert=ALL-UNNAMED --add-opens=java.base/java.nio.channels=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED --add-opens=java.base/javax.net.ssl=ALL-UNNAMED --add-opens=java.base/java.nio.file=ALL-UNNAMED --add-opens=java.base/java.time=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.management/java.lang.management=ALL-UNNAMED -Didea.test.cyclic.buffer.size=1048576 -javaagent:/Applications/IntelliJ IDEA.app/Contents/lib/idea_rt.jar=59517:/Applications/IntelliJ IDEA.app/Contents/bin -Dfile.encoding=UTF-8 -classpath /Applications/IntelliJ IDEA.app/Contents/lib/idea_rt.jar:/Applications/IntelliJ IDEA.app/Contents/plugins/junit/lib/junit5-rt.jar:/Applications/IntelliJ IDEA.app/Contents/plugins/junit/lib/junit-rt.jar:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/modules/reindex/out/test/classes:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/modules/reindex/out/test/resources:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/modules/reindex/out/production/classes:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/test/framework/out/production/classes:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/test/framework/out/production/resources:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/client/rest/out/production/classes:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/client/rest/out/production/resources:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/libs/ssl-config/out/production/classes:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/server/out/production/classes:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/server/out/production/resources:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/modules/parent-join/out/production/classes:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/modules/parent-join/out/production/resources:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/client/sniffer/out/production/classes:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/commons-codec/commons-codec/1.14/3cb1181b2141a7e752f5bdc998b7ef1849f726cf/commons-codec-1.14.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/commons-logging/commons-logging/1.1.3/f6f66e966c70a83ffbdb6f17a0919eaf7c8aca7f/commons-logging-1.1.3.jar:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/libs/cli/out/production/classes:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/modules/transport-netty4/out/production/classes:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/com.carrotsearch.randomizedtesting/randomizedtesting-runner/2.7.7/5f75b50ae052a15705806d66526ec0094e43d786/randomizedtesting-runner-2.7.7.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/junit/junit/4.12/2973d150c0dc1fefe998f834810d68f278ea58ec/junit-4.12.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest/2.1/9420ba32c29217b54eebd26ff7f9234d31c3fbb2/hamcrest-2.1.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-test-framework/9.0.0/e7c92c17f48acc7f94cf1dc261b0256605fcb059/lucene-test-framework-9.0.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-codecs/9.0.0/bdbf382f714babd1e7f5c11fe1e3743146652e4/lucene-codecs-9.0.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.mockito/mockito-core/4.0.0/f5195e0c4a45716bbd2d1d29173adbd148acce3a/mockito-core-4.0.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy/1.11.19/3ee8778b8e5287c57c9c3f3fd4c1c23e93e8779a/byte-buddy-1.11.19.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.objenesis/objenesis/3.2/7fadf57620c8b8abdf7519533e5527367cb51f09/objenesis-3.2.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.elasticsearch/mocksocket/1.2/190ad3f42fd6ab7367bae0c0e14d1dc4e4a3c361/mocksocket-1.2.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-core/2.10.4/8796585e716440d6dd5128b30359932a9eb74d0d/jackson-core-2.10.4.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/com.networknt/json-schema-validator/1.0.48/d28a6fb127101a59a452e338dbda973d09cf3434/json-schema-validator-1.0.48.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-databind/2.10.4/76e9152e93d4cf052f93a64596f633ba5b1c8ed9/jackson-databind-2.10.4.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-annotations/2.10.4/6ae6028aff033f194c9710ad87c224ccaadeed6c/jackson-annotations-2.10.4.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.commons/commons-compress/1.19/7e65777fb451ddab6a9c054beb879e521b7eab78/commons-compress-1.19.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.commons/commons-lang3/3.9/122c7cee69b53ed4a7681c03d4ee4c0e2765da5/commons-lang3-3.9.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.httpcomponents/httpclient/4.5.10/7ca2e4276f4ef95e4db725a8cd4a1d1e7585b9e5/httpclient-4.5.10.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.httpcomponents/httpcore/4.4.12/21ebaf6d532bc350ba95bd81938fa5f0e511c132/httpcore-4.4.12.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.httpcomponents/httpasyncclient/4.1.4/f3a3240681faae3fa46b573a4c7e50cec9db0d86/httpasyncclient-4.1.4.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.httpcomponents/httpcore-nio/4.4.12/84cd29eca842f31db02987cfedea245af020198b/httpcore-nio-4.4.12.jar:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/libs/core/out/production/classes:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/libs/x-content/out/production/classes:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/libs/lz4/out/production/classes:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/libs/secure-sm/out/production/classes:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/libs/geo/out/production/classes:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-core/9.0.0/be679fd274f264e4e8b02bc032d2788cd4076ab4/lucene-core-9.0.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-analysis-common/9.0.0/f78890829c3d6f15de48fdbc2c77ef4c0e3f005c/lucene-analysis-common-9.0.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-backward-codecs/9.0.0/9fb48d0244799e18299449ee62459caab0728490/lucene-backward-codecs-9.0.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-grouping/9.0.0/27ebe235d427b4e392fabab9b6bfa09524ca7f8b/lucene-grouping-9.0.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-highlighter/9.0.0/a3cb395c2e8c672e6eec951b2b02371a4a883f73/lucene-highlighter-9.0.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-join/9.0.0/94a855b5d09a6601289aeaeba0f11d5539552590/lucene-join-9.0.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-memory/9.0.0/2371c95031422bc1f501d43ffcc7311baed4b35b/lucene-memory-9.0.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-misc/9.0.0/25c6170f4fa2f707908dfb92fbafc76727f901e0/lucene-misc-9.0.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-queries/9.0.0/87b4c7833d30895baf7091f9cb0db878e970b604/lucene-queries-9.0.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-queryparser/9.0.0/bf13395ad2033bca3182fcbc83204e8ae1951945/lucene-queryparser-9.0.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-sandbox/9.0.0/3c153a1dc1da3f98083cc932c9476df4b77b0ca5/lucene-sandbox-9.0.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-spatial3d/9.0.0/6b4ee47f218ed3d123c1b07671677a2e4f3c133b/lucene-spatial3d-9.0.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-suggest/9.0.0/a7d0e7279737114c039f5214082da948732096a6/lucene-suggest-9.0.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/com.carrotsearch/hppc/0.8.1/ffc7ba8f289428b9508ab484b8001dea944ae603/hppc-0.8.1.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/com.tdunning/t-digest/3.2/2ab94758b0276a8a26102adf8d528cf6d0567b9a/t-digest-3.2.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.hdrhistogram/HdrHistogram/2.1.9/e4631ce165eb400edecfa32e03d3f1be53dee754/HdrHistogram-2.1.9.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.logging.log4j/log4j-api/2.17.1/d771af8e336e372fb5399c99edabe0919aeaf5b2/log4j-api-2.17.1.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.apache.logging.log4j/log4j-core/2.17.1/779f60f3844dadc3ef597976fcb1e5127b1f343d/log4j-core-2.17.1.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/net.java.dev.jna/jna/5.10.0/7cf4c87dd802db50721db66947aa237d7ad09418/jna-5.10.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/co.elastic.logging/log4j2-ecs-layout/1.2.0/ba51fb2064cd5f6bc136e95c1463e3e68d823403/log4j2-ecs-layout-1.2.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/co.elastic.logging/ecs-logging-core/1.2.0/37a47ec302777aca9f8ea10b7316f3a79d5d6202/ecs-logging-core-1.2.0.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/net.sf.jopt-simple/jopt-simple/5.0.2/98cafc6081d5632b61be2c9e60650b64ddbc637c/jopt-simple-5.0.2.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/io.netty/netty-buffer/4.1.73.Final/244a569c9aae973f6f485ac9801d79c1eca36daa/netty-buffer-4.1.73.Final.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/io.netty/netty-codec/4.1.73.Final/9496a30a349863a4c6fa10d5c36b4f3b495d3a31/netty-codec-4.1.73.Final.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/io.netty/netty-codec-http/4.1.73.Final/1ceeac4429b9bd517dc05e376a144bbe6b6bd038/netty-codec-http-4.1.73.Final.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/io.netty/netty-common/4.1.73.Final/27731b58d741b6faa6a00fa3285e7a55cc47be01/netty-common-4.1.73.Final.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/io.netty/netty-handler/4.1.73.Final/1a2231c0074f88254865c3769a4b5842939ea04d/netty-handler-4.1.73.Final.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/io.netty/netty-resolver/4.1.73.Final/bfe83710f0c1739019613e81a06101020ca65def/netty-resolver-4.1.73.Final.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/io.netty/netty-transport/4.1.73.Final/abb155ddff196ccedfe85b810d4b9375ef85fcfa/netty-transport-4.1.73.Final.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.yaml/snakeyaml/1.26/a78a8747147d2c5807683e76ec2b633e95c14fe9/snakeyaml-1.26.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.dataformat/jackson-dataformat-smile/2.10.4/c872c2e224cfdcc5481037d477f5890f05c001b4/jackson-dataformat-smile-2.10.4.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.dataformat/jackson-dataformat-yaml/2.10.4/8a7f3c6b640bd89214807af6d8160b4b3b16af93/jackson-dataformat-yaml-2.10.4.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.dataformat/jackson-dataformat-cbor/2.10.4/c854bb2d46138198cb5d4aae86ef6c04b8bc1e70/jackson-dataformat-cbor-2.10.4.jar:/Users/przemyslawgomulka/.gradle/caches/modules-2/files-2.1/org.lz4/lz4-java/1.8.0/4b986a99445e49ea5fbf5d149c4b63f6ed6c6780/lz4-java-1.8.0.jar:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/libs/plugin-classloader/out/production/classes:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/modules/reindex/build/generated-resources:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/modules/reindex/build/restResources/yamlSpecs:/Users/przemyslawgomulka/workspace/pgomulka/elasticsearch/modules/reindex/build/generated-test-resources/parent-join com.intellij.rt.junit.JUnitStarter -ideVersion5 -junit4 org.elasticsearch.migration.SystemIndexMigrationIT
[2022-02-22T16:25:49,241][INFO ][o.e.m.SystemIndexMigrationIT] [testSystemIndexMigrationCanBeInterruptedWithShutdown] before test
[2022-02-22T16:25:49,245][INFO ][o.e.m.SystemIndexMigrationIT] [testSystemIndexMigrationCanBeInterruptedWithShutdown] [SystemIndexMigrationIT#testSystemIndexMigrationCanBeInterruptedWithShutdown]: setting up test
[2022-02-22T16:25:49,269][INFO ][o.e.t.InternalTestCluster] [testSystemIndexMigrationCanBeInterruptedWithShutdown] Setup InternalTestCluster [TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[-5868684473565347403]-HASH=[A5535B3C13D4]-cluster] with seed [AE8E3E909E4981B5] using [0] dedicated masters, [0] (data) nodes and [0] coord only nodes (master nodes are [manual])
[2022-02-22T16:25:49,307][INFO ][o.e.m.SystemIndexMigrationIT] [testSystemIndexMigrationCanBeInterruptedWithShutdown] [SystemIndexMigrationIT#testSystemIndexMigrationCanBeInterruptedWithShutdown]: all set up test
[2022-02-22T16:25:49,362][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] version[8.2.0], pid[64094], build[unknown/unknown/unknown/unknown], OS[Mac OS X/12.2.1/aarch64], JVM[Oracle Corporation/Java HotSpot(TM) 64-Bit Server VM/17.0.2/17.0.2+8-LTS-86]
[2022-02-22T16:25:49,362][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] JVM home [/Library/Java/JavaVirtualMachines/jdk-17.0.2.jdk/Contents/Home]
[2022-02-22T16:25:49,364][WARN ][o.e.d.n.Node             ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] data_stream.dataset="deprecation.elasticsearch" data_stream.namespace="default" data_stream.type="logs" elasticsearch.event.category="other" event.code="no-jdk" message="no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release"
[2022-02-22T16:25:49,364][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] JVM arguments [-ea, -Djava.locale.providers=SPI,COMPAT, --add-opens=java.base/java.security.cert=ALL-UNNAMED, --add-opens=java.base/java.nio.channels=ALL-UNNAMED, --add-opens=java.base/java.net=ALL-UNNAMED, --add-opens=java.base/javax.net.ssl=ALL-UNNAMED, --add-opens=java.base/java.nio.file=ALL-UNNAMED, --add-opens=java.base/java.time=ALL-UNNAMED, --add-opens=java.base/java.lang=ALL-UNNAMED, --add-opens=java.management/java.lang.management=ALL-UNNAMED, -Didea.test.cyclic.buffer.size=1048576, -javaagent:/Applications/IntelliJ IDEA.app/Contents/lib/idea_rt.jar=59517:/Applications/IntelliJ IDEA.app/Contents/bin, -Dfile.encoding=UTF-8]
[2022-02-22T16:25:49,364][WARN ][o.e.d.n.Node             ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] data_stream.dataset="deprecation.elasticsearch" data_stream.namespace="default" data_stream.type="logs" elasticsearch.event.category="settings" event.code="shared-data-path" message="setting [path.shared_data] is deprecated and will be removed in a future release"
[2022-02-22T16:25:49,368][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] no modules loaded
[2022-02-22T16:25:49,368][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.migration.AbstractFeatureMigrationIntegTest$TestPlugin]
[2022-02-22T16:25:49,369][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.reindex.ReindexPlugin]
[2022-02-22T16:25:49,369][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.script.MockScriptService$TestPlugin]
[2022-02-22T16:25:49,369][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.test.ESIntegTestCase$AssertActionNamePlugin]
[2022-02-22T16:25:49,369][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.test.ESIntegTestCase$TestSeedPlugin]
[2022-02-22T16:25:49,369][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.test.MockHttpTransport$TestPlugin]
[2022-02-22T16:25:49,369][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.test.TestGeoShapeFieldMapperPlugin]
[2022-02-22T16:25:49,369][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.transport.netty4.Netty4Plugin]
[2022-02-22T16:25:49,390][INFO ][o.e.e.NodeEnvironment    ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] using [1] data paths, mounts [[/System/Volumes/Data (/dev/disk3s5)]], net usable_space [812.5gb], net total_space [926.3gb], types [apfs]
[2022-02-22T16:25:49,390][INFO ][o.e.e.NodeEnvironment    ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] heap size [16gb], compressed ordinary object pointers [true]
[2022-02-22T16:25:49,412][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] node name [node_t0], node ID [vvb_ksGqTQ6iTJrASggbnw], cluster name [TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[-5868684473565347403]-HASH=[A5535B3C13D4]-cluster], roles [master]
[2022-02-22T16:25:49,450][INFO ][i.n.u.i.PlatformDependent] [testSystemIndexMigrationCanBeInterruptedWithShutdown] Your platform does not provide complete low-level API for accessing direct buffers reliably. Unless explicitly requested, heap buffer will always be preferred to avoid potential system instability.
[2022-02-22T16:25:50,377][INFO ][o.e.t.n.NettyAllocator   ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] creating NettyAllocator with the following configs: [name=elasticsearch_configured, chunk_size=1mb, suggested_max_allocation_size=1mb, factors={es.unsafe.use_netty_default_chunk_and_page_size=false, g1gc_enabled=true, g1gc_region_size=8mb}]
[2022-02-22T16:25:50,397][INFO ][o.e.i.r.RecoverySettings ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] using rate limit [102mb] with [default=102mb, read=0b, write=0b, max=0b]
[2022-02-22T16:25:50,418][INFO ][o.e.d.DiscoveryModule    ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] using discovery type [multi-node] and seed hosts providers [settings, file]
[2022-02-22T16:25:50,688][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] initialized
[2022-02-22T16:25:50,690][INFO ][o.e.n.Node               ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[-5868684473565347403]-HASH=[A5535B3C13D4]-cluster[T#1]]] starting ...
[2022-02-22T16:25:50,758][INFO ][o.e.t.TransportService   ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[-5868684473565347403]-HASH=[A5535B3C13D4]-cluster[T#1]]] publish_address {127.0.0.1:59521}, bound_addresses {[::1]:59520}, {127.0.0.1:59521}
[2022-02-22T16:25:50,844][INFO ][o.e.c.c.Coordinator      ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[-5868684473565347403]-HASH=[A5535B3C13D4]-cluster[T#1]]] setting initial configuration to VotingConfiguration{vvb_ksGqTQ6iTJrASggbnw}
[2022-02-22T16:25:50,965][INFO ][o.e.c.s.MasterService    ] [node_t0] elected-as-master ([1] nodes joined)[_FINISH_ELECTION_, {node_t0}{vvb_ksGqTQ6iTJrASggbnw}{oS7cOM2YR4WhH0rvPZxNnw}{127.0.0.1}{127.0.0.1:59521}{m} completing election], term: 1, version: 1, delta: master node changed {previous [], current [{node_t0}{vvb_ksGqTQ6iTJrASggbnw}{oS7cOM2YR4WhH0rvPZxNnw}{127.0.0.1}{127.0.0.1:59521}{m}]}
[2022-02-22T16:25:50,995][INFO ][o.e.c.c.CoordinationState] [node_t0] cluster UUID set to [NGLyD5XQSvWnPPuueN7G8w]
[2022-02-22T16:25:51,007][INFO ][o.e.c.s.ClusterApplierService] [node_t0] master node changed {previous [], current [{node_t0}{vvb_ksGqTQ6iTJrASggbnw}{oS7cOM2YR4WhH0rvPZxNnw}{127.0.0.1}{127.0.0.1:59521}{m}]}, term: 1, version: 1, reason: Publication{term=1, version=1}
[2022-02-22T16:25:51,058][INFO ][o.e.n.Node               ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[-5868684473565347403]-HASH=[A5535B3C13D4]-cluster[T#1]]] started
[2022-02-22T16:25:51,060][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] version[8.2.0], pid[64094], build[unknown/unknown/unknown/unknown], OS[Mac OS X/12.2.1/aarch64], JVM[Oracle Corporation/Java HotSpot(TM) 64-Bit Server VM/17.0.2/17.0.2+8-LTS-86]
[2022-02-22T16:25:51,060][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] JVM home [/Library/Java/JavaVirtualMachines/jdk-17.0.2.jdk/Contents/Home]
[2022-02-22T16:25:51,060][WARN ][o.e.d.n.Node             ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] data_stream.dataset="deprecation.elasticsearch" data_stream.namespace="default" data_stream.type="logs" elasticsearch.event.category="other" event.code="no-jdk" message="no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release"
[2022-02-22T16:25:51,060][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] JVM arguments [-ea, -Djava.locale.providers=SPI,COMPAT, --add-opens=java.base/java.security.cert=ALL-UNNAMED, --add-opens=java.base/java.nio.channels=ALL-UNNAMED, --add-opens=java.base/java.net=ALL-UNNAMED, --add-opens=java.base/javax.net.ssl=ALL-UNNAMED, --add-opens=java.base/java.nio.file=ALL-UNNAMED, --add-opens=java.base/java.time=ALL-UNNAMED, --add-opens=java.base/java.lang=ALL-UNNAMED, --add-opens=java.management/java.lang.management=ALL-UNNAMED, -Didea.test.cyclic.buffer.size=1048576, -javaagent:/Applications/IntelliJ IDEA.app/Contents/lib/idea_rt.jar=59517:/Applications/IntelliJ IDEA.app/Contents/bin, -Dfile.encoding=UTF-8]
[2022-02-22T16:25:51,060][WARN ][o.e.d.n.Node             ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] data_stream.dataset="deprecation.elasticsearch" data_stream.namespace="default" data_stream.type="logs" elasticsearch.event.category="settings" event.code="shared-data-path" message="setting [path.shared_data] is deprecated and will be removed in a future release"
[2022-02-22T16:25:51,061][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] no modules loaded
[2022-02-22T16:25:51,061][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.migration.AbstractFeatureMigrationIntegTest$TestPlugin]
[2022-02-22T16:25:51,061][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.reindex.ReindexPlugin]
[2022-02-22T16:25:51,061][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.script.MockScriptService$TestPlugin]
[2022-02-22T16:25:51,061][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.test.ESIntegTestCase$AssertActionNamePlugin]
[2022-02-22T16:25:51,061][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.test.ESIntegTestCase$TestSeedPlugin]
[2022-02-22T16:25:51,061][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.test.MockHttpTransport$TestPlugin]
[2022-02-22T16:25:51,061][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.test.TestGeoShapeFieldMapperPlugin]
[2022-02-22T16:25:51,061][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.transport.netty4.Netty4Plugin]
[2022-02-22T16:25:51,062][INFO ][o.e.e.NodeEnvironment    ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] using [1] data paths, mounts [[/System/Volumes/Data (/dev/disk3s5)]], net usable_space [812.5gb], net total_space [926.3gb], types [apfs]
[2022-02-22T16:25:51,062][INFO ][o.e.e.NodeEnvironment    ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] heap size [16gb], compressed ordinary object pointers [true]
[2022-02-22T16:25:51,064][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] node name [node_t1], node ID [ntz6Qb7ZQvusv1DZfmoRkg], cluster name [TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[-5868684473565347403]-HASH=[A5535B3C13D4]-cluster], roles [data_hot, ml, data_frozen, ingest, data_cold, data, remote_cluster_client, master, data_warm, data_content, transform]
[2022-02-22T16:25:51,085][INFO ][o.e.g.GatewayService     ] [node_t0] recovered [0] indices into cluster_state
[2022-02-22T16:25:51,087][INFO ][o.e.i.r.RecoverySettings ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] using rate limit [40mb] with [default=40mb, read=0b, write=0b, max=0b]
[2022-02-22T16:25:51,088][INFO ][o.e.d.DiscoveryModule    ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] using discovery type [multi-node] and seed hosts providers [settings, file]
[2022-02-22T16:25:51,102][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] initialized
[2022-02-22T16:25:51,103][INFO ][o.e.n.Node               ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[-5868684473565347403]-HASH=[A5535B3C13D4]-cluster[T#2]]] starting ...
[2022-02-22T16:25:51,106][INFO ][o.e.t.TransportService   ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[-5868684473565347403]-HASH=[A5535B3C13D4]-cluster[T#2]]] publish_address {127.0.0.1:59523}, bound_addresses {[::1]:59522}, {127.0.0.1:59523}
[2022-02-22T16:25:51,180][INFO ][o.e.c.s.MasterService    ] [node_t0] node-join[{node_t1}{ntz6Qb7ZQvusv1DZfmoRkg}{WZ4CG0fzSiOCFwEtg-7DUQ}{127.0.0.1}{127.0.0.1:59523}{cdfhilmrstw} joining], term: 1, version: 3, delta: added {{node_t1}{ntz6Qb7ZQvusv1DZfmoRkg}{WZ4CG0fzSiOCFwEtg-7DUQ}{127.0.0.1}{127.0.0.1:59523}{cdfhilmrstw}}
[2022-02-22T16:25:51,201][INFO ][o.e.c.s.ClusterApplierService] [node_t1] master node changed {previous [], current [{node_t0}{vvb_ksGqTQ6iTJrASggbnw}{oS7cOM2YR4WhH0rvPZxNnw}{127.0.0.1}{127.0.0.1:59521}{m}]}, added {{node_t0}{vvb_ksGqTQ6iTJrASggbnw}{oS7cOM2YR4WhH0rvPZxNnw}{127.0.0.1}{127.0.0.1:59521}{m}}, term: 1, version: 3, reason: ApplyCommitRequest{term=1, version=3, sourceNode={node_t0}{vvb_ksGqTQ6iTJrASggbnw}{oS7cOM2YR4WhH0rvPZxNnw}{127.0.0.1}{127.0.0.1:59521}{m}}
[2022-02-22T16:25:51,203][INFO ][o.e.n.Node               ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[-5868684473565347403]-HASH=[A5535B3C13D4]-cluster[T#2]]] started
[2022-02-22T16:25:51,204][INFO ][o.e.c.s.ClusterApplierService] [node_t0] added {{node_t1}{ntz6Qb7ZQvusv1DZfmoRkg}{WZ4CG0fzSiOCFwEtg-7DUQ}{127.0.0.1}{127.0.0.1:59523}{cdfhilmrstw}}, term: 1, version: 3, reason: Publication{term=1, version=3}
[2022-02-22T16:25:51,205][INFO ][o.e.m.SystemIndexMigrationIT] [testSystemIndexMigrationCanBeInterruptedWithShutdown] using custom data_path for index: [plNQDfqpfo]
[2022-02-22T16:25:51,317][INFO ][o.e.c.m.MetadataCreateIndexService] [node_t0] [.int-man-old] creating index, cause [api], templates [], shards [1]/[0]
[2022-02-22T16:25:51,436][INFO ][o.e.c.r.a.AllocationService] [node_t0] current.health="GREEN" message="Cluster health status changed from [YELLOW] to [GREEN] (reason: [shards started [[.int-man-old][0]]])." previous.health="YELLOW" reason="shards started [[.int-man-old][0]]"
[2022-02-22T16:25:51,476][INFO ][o.e.m.SystemIndexMigrationIT] [testSystemIndexMigrationCanBeInterruptedWithShutdown] Index [100] docs async: [true] bulk: [false]
[2022-02-22T16:25:51,667][INFO ][o.e.m.SystemIndexMigrationIT] [testSystemIndexMigrationCanBeInterruptedWithShutdown] migrate feature api called
[2022-02-22T16:25:51,685][INFO ][o.e.m.SystemIndexMigrationIT] [node_t0] Task created
[2022-02-22T16:25:51,685][INFO ][o.e.t.InternalTestCluster] [testSystemIndexMigrationCanBeInterruptedWithShutdown] Restarting node [node_t1] 
[2022-02-22T16:25:51,685][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] stopping ...
[2022-02-22T16:25:51,687][INFO ][o.e.c.c.Coordinator      ] [node_t1] master node [{node_t0}{vvb_ksGqTQ6iTJrASggbnw}{oS7cOM2YR4WhH0rvPZxNnw}{127.0.0.1}{127.0.0.1:59521}{m}] disconnected, restarting discovery
[2022-02-22T16:25:51,689][INFO ][o.e.t.ClusterConnectionManager] [node_t0] transport connection to [{node_t1}{ntz6Qb7ZQvusv1DZfmoRkg}{WZ4CG0fzSiOCFwEtg-7DUQ}{127.0.0.1}{127.0.0.1:59523}{cdfhilmrstw}] closed by remote
[2022-02-22T16:25:51,689][WARN ][o.e.d.HandshakingTransportAddressConnector] [node_t1] handshake failed for [connectToRemoteMasterNode[127.0.0.1:59521]]
org.elasticsearch.transport.SendRequestTransportException: [][127.0.0.1:59521][internal:transport/handshake]
	at org.elasticsearch.transport.TransportService.sendRequestInternal(TransportService.java:859) [classes/:?]
	at org.elasticsearch.transport.TransportService.sendRequest(TransportService.java:745) [classes/:?]
	at org.elasticsearch.transport.TransportService.handshake(TransportService.java:469) [classes/:?]
	at org.elasticsearch.transport.TransportService.handshake(TransportService.java:446) [classes/:?]
	at org.elasticsearch.discovery.HandshakingTransportAddressConnector$1.lambda$doRun$0(HandshakingTransportAddressConnector.java:105) [classes/:?]
	at org.elasticsearch.action.ActionListener$DelegatingFailureActionListener.onResponse(ActionListener.java:219) [classes/:?]
	at org.elasticsearch.action.ActionListener$MappedActionListener.onResponse(ActionListener.java:101) [classes/:?]
	at org.elasticsearch.action.support.ThreadedActionListener$1.doRun(ThreadedActionListener.java:52) [classes/:?]
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:776) [classes/:?]
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26) [classes/:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: org.elasticsearch.node.NodeClosedException: node closed {node_t1}{ntz6Qb7ZQvusv1DZfmoRkg}{WZ4CG0fzSiOCFwEtg-7DUQ}{127.0.0.1}{127.0.0.1:59523}{cdfhilmrstw}
	at org.elasticsearch.transport.TransportService.sendRequestInternal(TransportService.java:840) ~[classes/:?]
	... 12 more
[2022-02-22T16:25:51,696][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] stopped
[2022-02-22T16:25:51,696][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] closing ...
[2022-02-22T16:25:51,699][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] closed
[2022-02-22T16:25:51,700][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] version[8.2.0], pid[64094], build[unknown/unknown/unknown/unknown], OS[Mac OS X/12.2.1/aarch64], JVM[Oracle Corporation/Java HotSpot(TM) 64-Bit Server VM/17.0.2/17.0.2+8-LTS-86]
[2022-02-22T16:25:51,700][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] JVM home [/Library/Java/JavaVirtualMachines/jdk-17.0.2.jdk/Contents/Home]
[2022-02-22T16:25:51,700][WARN ][o.e.d.n.Node             ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] data_stream.dataset="deprecation.elasticsearch" data_stream.namespace="default" data_stream.type="logs" elasticsearch.event.category="other" event.code="no-jdk" message="no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release"
[2022-02-22T16:25:51,700][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] JVM arguments [-ea, -Djava.locale.providers=SPI,COMPAT, --add-opens=java.base/java.security.cert=ALL-UNNAMED, --add-opens=java.base/java.nio.channels=ALL-UNNAMED, --add-opens=java.base/java.net=ALL-UNNAMED, --add-opens=java.base/javax.net.ssl=ALL-UNNAMED, --add-opens=java.base/java.nio.file=ALL-UNNAMED, --add-opens=java.base/java.time=ALL-UNNAMED, --add-opens=java.base/java.lang=ALL-UNNAMED, --add-opens=java.management/java.lang.management=ALL-UNNAMED, -Didea.test.cyclic.buffer.size=1048576, -javaagent:/Applications/IntelliJ IDEA.app/Contents/lib/idea_rt.jar=59517:/Applications/IntelliJ IDEA.app/Contents/bin, -Dfile.encoding=UTF-8]
[2022-02-22T16:25:51,700][WARN ][o.e.d.n.Node             ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] data_stream.dataset="deprecation.elasticsearch" data_stream.namespace="default" data_stream.type="logs" elasticsearch.event.category="settings" event.code="shared-data-path" message="setting [path.shared_data] is deprecated and will be removed in a future release"
[2022-02-22T16:25:51,701][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] no modules loaded
[2022-02-22T16:25:51,701][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.migration.AbstractFeatureMigrationIntegTest$TestPlugin]
[2022-02-22T16:25:51,701][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.reindex.ReindexPlugin]
[2022-02-22T16:25:51,701][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.script.MockScriptService$TestPlugin]
[2022-02-22T16:25:51,701][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.test.ESIntegTestCase$AssertActionNamePlugin]
[2022-02-22T16:25:51,701][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.test.ESIntegTestCase$TestSeedPlugin]
[2022-02-22T16:25:51,701][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.test.MockHttpTransport$TestPlugin]
[2022-02-22T16:25:51,701][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.test.TestGeoShapeFieldMapperPlugin]
[2022-02-22T16:25:51,701][INFO ][o.e.p.PluginsService     ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] loaded plugin [org.elasticsearch.transport.netty4.Netty4Plugin]
[2022-02-22T16:25:51,702][INFO ][o.e.e.NodeEnvironment    ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] using [1] data paths, mounts [[/System/Volumes/Data (/dev/disk3s5)]], net usable_space [812.5gb], net total_space [926.3gb], types [apfs]
[2022-02-22T16:25:51,702][INFO ][o.e.c.r.a.AllocationService] [node_t0] current.health="RED" message="Cluster health status changed from [GREEN] to [RED] (reason: [{node_t1}{ntz6Qb7ZQvusv1DZfmoRkg}{WZ4CG0fzSiOCFwEtg-7DUQ}{127.0.0.1}{127.0.0.1:59523}{cdfhilmrstw} reason: disconnected])." previous.health="GREEN" reason="{node_t1}{ntz6Qb7ZQvusv1DZfmoRkg}{WZ4CG0fzSiOCFwEtg-7DUQ}{127.0.0.1}{127.0.0.1:59523}{cdfhilmrstw} reason: disconnected"
[2022-02-22T16:25:51,702][INFO ][o.e.e.NodeEnvironment    ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] heap size [16gb], compressed ordinary object pointers [true]
[2022-02-22T16:25:51,704][INFO ][o.e.c.s.MasterService    ] [node_t0] node-left[{node_t1}{ntz6Qb7ZQvusv1DZfmoRkg}{WZ4CG0fzSiOCFwEtg-7DUQ}{127.0.0.1}{127.0.0.1:59523}{cdfhilmrstw} reason: disconnected], term: 1, version: 7, delta: removed {{node_t1}{ntz6Qb7ZQvusv1DZfmoRkg}{WZ4CG0fzSiOCFwEtg-7DUQ}{127.0.0.1}{127.0.0.1:59523}{cdfhilmrstw}}
[2022-02-22T16:25:51,705][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] node name [node_t1], node ID [ntz6Qb7ZQvusv1DZfmoRkg], cluster name [TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[-5868684473565347403]-HASH=[A5535B3C13D4]-cluster], roles [data_hot, ml, data_frozen, ingest, data_cold, data, remote_cluster_client, master, data_warm, data_content, transform]
[2022-02-22T16:25:51,711][INFO ][o.e.c.s.ClusterApplierService] [node_t0] removed {{node_t1}{ntz6Qb7ZQvusv1DZfmoRkg}{WZ4CG0fzSiOCFwEtg-7DUQ}{127.0.0.1}{127.0.0.1:59523}{cdfhilmrstw}}, term: 1, version: 7, reason: Publication{term=1, version=7}
[2022-02-22T16:25:51,714][INFO ][o.e.c.r.DelayedAllocationService] [node_t0] scheduling reroute for delayed shards in [59.9s] (1 delayed shards)
[2022-02-22T16:25:51,715][INFO ][o.e.u.SystemIndexMigrator] [node_t0] preparing to migrate old index [.int-man-old] from feature [A-test-feature] to new index [.int-man-old-reindexed-for-8]
[2022-02-22T16:25:51,724][INFO ][o.e.u.SystemIndexMigrator] [node_t0] migrating index [.int-man-old] from feature [A-test-feature] to new index [.int-man-old-reindexed-for-8]
[2022-02-22T16:25:51,728][INFO ][o.e.c.m.MetadataCreateIndexService] [node_t0] [.int-man-old-reindexed-for-8] creating index, cause [migrate-system-index], templates [], shards [1]/[0]
[2022-02-22T16:25:51,735][INFO ][o.e.i.r.RecoverySettings ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] using rate limit [40mb] with [default=40mb, read=0b, write=0b, max=0b]
[2022-02-22T16:25:51,736][INFO ][o.e.d.DiscoveryModule    ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] using discovery type [multi-node] and seed hosts providers [settings, file]
[2022-02-22T16:25:51,750][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] initialized
[2022-02-22T16:25:51,750][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] starting ...
[2022-02-22T16:25:51,753][INFO ][o.e.t.TransportService   ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] publish_address {127.0.0.1:59546}, bound_addresses {[::1]:59545}, {127.0.0.1:59546}
[2022-02-22T16:25:51,779][ERROR][o.e.u.SystemIndexMigrator] [node_t0] error occurred while reindexing index [.int-man-old] from feature [A-test-feature] to destination index [.int-man-old-reindexed-for-8]
org.elasticsearch.action.search.SearchPhaseExecutionException: 
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.onPhaseFailure(AbstractSearchAsyncAction.java:716) [classes/:?]
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.executePhase(AbstractSearchAsyncAction.java:462) [classes/:?]
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.start(AbstractSearchAsyncAction.java:202) [classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.executeSearch(TransportSearchAction.java:1031) [classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.executeLocalSearch(TransportSearchAction.java:747) [classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.lambda$executeRequest$6(TransportSearchAction.java:390) [classes/:?]
	at org.elasticsearch.action.ActionListener$1.onResponse(ActionListener.java:136) [classes/:?]
	at org.elasticsearch.index.query.Rewriteable.rewriteAndFetch(Rewriteable.java:112) [classes/:?]
	at org.elasticsearch.index.query.Rewriteable.rewriteAndFetch(Rewriteable.java:77) [classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.executeRequest(TransportSearchAction.java:478) [classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.doExecute(TransportSearchAction.java:277) [classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.doExecute(TransportSearchAction.java:103) [classes/:?]
	at org.elasticsearch.action.support.TransportAction$RequestFilterChain.proceed(TransportAction.java:79) [classes/:?]
	at org.elasticsearch.action.support.TransportAction.execute(TransportAction.java:54) [classes/:?]
	at org.elasticsearch.tasks.TaskManager.registerAndExecute(TaskManager.java:170) [classes/:?]
	at org.elasticsearch.client.internal.node.NodeClient.executeLocally(NodeClient.java:113) [classes/:?]
	at org.elasticsearch.client.internal.node.NodeClient.doExecute(NodeClient.java:91) [classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.execute(AbstractClient.java:380) [classes/:?]
	at org.elasticsearch.client.internal.FilterClient.doExecute(FilterClient.java:57) [classes/:?]
	at org.elasticsearch.client.internal.ParentTaskAssigningClient.doExecute(ParentTaskAssigningClient.java:55) [classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.execute(AbstractClient.java:380) [classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.search(AbstractClient.java:515) [classes/:?]
	at org.elasticsearch.index.reindex.ClientScrollableHitSource.doStart(ClientScrollableHitSource.java:74) [classes/:?]
	at org.elasticsearch.index.reindex.ScrollableHitSource.start(ScrollableHitSource.java:73) [classes/:?]
	at org.elasticsearch.reindex.AbstractAsyncBulkByScrollAction.start(AbstractAsyncBulkByScrollAction.java:327) [classes/:?]
	at org.elasticsearch.reindex.Reindexer.lambda$execute$0(Reindexer.java:119) [classes/:?]
	at org.elasticsearch.reindex.BulkByScrollParallelizationHelper.executeSlicedAction(BulkByScrollParallelizationHelper.java:95) [classes/:?]
	at org.elasticsearch.reindex.Reindexer.execute(Reindexer.java:98) [classes/:?]
	at org.elasticsearch.reindex.TransportReindexAction.lambda$doExecute$0(TransportReindexAction.java:103) [classes/:?]
	at org.elasticsearch.action.ActionListener$DelegatingFailureActionListener.onResponse(ActionListener.java:219) [classes/:?]
	at org.elasticsearch.reindex.BulkByScrollParallelizationHelper.initTaskState(BulkByScrollParallelizationHelper.java:125) [classes/:?]
	at org.elasticsearch.reindex.Reindexer.initTask(Reindexer.java:94) [classes/:?]
	at org.elasticsearch.reindex.TransportReindexAction.doExecute(TransportReindexAction.java:100) [classes/:?]
	at org.elasticsearch.reindex.TransportReindexAction.doExecute(TransportReindexAction.java:36) [classes/:?]
	at org.elasticsearch.action.support.TransportAction$RequestFilterChain.proceed(TransportAction.java:79) [classes/:?]
	at org.elasticsearch.action.support.TransportAction.execute(TransportAction.java:54) [classes/:?]
	at org.elasticsearch.tasks.TaskManager.registerAndExecute(TaskManager.java:170) [classes/:?]
	at org.elasticsearch.client.internal.node.NodeClient.executeLocally(NodeClient.java:113) [classes/:?]
	at org.elasticsearch.client.internal.node.NodeClient.doExecute(NodeClient.java:91) [classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.execute(AbstractClient.java:380) [classes/:?]
	at org.elasticsearch.client.internal.FilterClient.doExecute(FilterClient.java:57) [classes/:?]
	at org.elasticsearch.client.internal.ParentTaskAssigningClient.doExecute(ParentTaskAssigningClient.java:55) [classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.execute(AbstractClient.java:380) [classes/:?]
	at org.elasticsearch.client.internal.FilterClient.doExecute(FilterClient.java:57) [classes/:?]
	at org.elasticsearch.client.internal.OriginSettingClient.doExecute(OriginSettingClient.java:43) [classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.execute(AbstractClient.java:380) [classes/:?]
	at org.elasticsearch.upgrades.SystemIndexMigrator.reindex(SystemIndexMigrator.java:523) [classes/:?]
	at org.elasticsearch.upgrades.SystemIndexMigrator.lambda$migrateSingleIndex$14(SystemIndexMigrator.java:402) [classes/:?]
	at org.elasticsearch.action.ActionListener$1.onResponse(ActionListener.java:136) [classes/:?]
	at org.elasticsearch.action.support.ContextPreservingActionListener.onResponse(ContextPreservingActionListener.java:31) [classes/:?]
	at org.elasticsearch.cluster.AckedClusterStateUpdateTask.onAckTimeout(AckedClusterStateUpdateTask.java:71) [classes/:?]
	at org.elasticsearch.cluster.service.MasterService$ContextPreservingAckListener.onAckTimeout(MasterService.java:665) [classes/:?]
	at org.elasticsearch.cluster.service.MasterService$TaskAckListener.onTimeout(MasterService.java:765) [classes/:?]
	at org.elasticsearch.cluster.service.MasterService$TaskAckListener.onCommit(MasterService.java:719) [classes/:?]
	at org.elasticsearch.cluster.service.MasterService$CompositeTaskAckListener.onCommit(MasterService.java:778) [classes/:?]
	at org.elasticsearch.cluster.coordination.Coordinator$CoordinatorPublication$1.onCommit(Coordinator.java:1545) [classes/:?]
	at org.elasticsearch.cluster.coordination.Publication$PublicationTarget.lambda$handlePublishResponse$0(Publication.java:268) [classes/:?]
	at java.util.Optional.ifPresent(Optional.java:178) [?:?]
	at org.elasticsearch.cluster.coordination.Publication$PublicationTarget.handlePublishResponse(Publication.java:265) [classes/:?]
	at org.elasticsearch.cluster.coordination.Publication$PublicationTarget$PublishResponseHandler.onResponse(Publication.java:364) [classes/:?]
	at org.elasticsearch.cluster.coordination.Publication$PublicationTarget$PublishResponseHandler.onResponse(Publication.java:341) [classes/:?]
	at org.elasticsearch.cluster.coordination.Coordinator$4.onResponse(Coordinator.java:1366) [classes/:?]
	at org.elasticsearch.cluster.coordination.PublicationTransportHandler$PublicationContext$1.onResponse(PublicationTransportHandler.java:354) [classes/:?]
	at org.elasticsearch.cluster.coordination.PublicationTransportHandler$PublicationContext$1.onResponse(PublicationTransportHandler.java:350) [classes/:?]
	at org.elasticsearch.action.ActionListener$DelegatingActionListener.onResponse(ActionListener.java:186) [classes/:?]
	at org.elasticsearch.action.ActionListener$RunAfterActionListener.onResponse(ActionListener.java:341) [classes/:?]
	at org.elasticsearch.action.ActionListener$RunAfterActionListener.onResponse(ActionListener.java:341) [classes/:?]
	at org.elasticsearch.action.ActionListenerResponseHandler.handleResponse(ActionListenerResponseHandler.java:43) [classes/:?]
	at org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleResponse(TransportService.java:1339) [classes/:?]
	at org.elasticsearch.transport.TransportService$DirectResponseChannel.processResponse(TransportService.java:1417) [classes/:?]
	at org.elasticsearch.transport.TransportService$DirectResponseChannel$1.run(TransportService.java:1402) [classes/:?]
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:717) [classes/:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: org.elasticsearch.action.search.SearchPhaseExecutionException: Search rejected due to missing shards [[.int-man-old][0]]. Consider using `allow_partial_search_results` setting to bypass this error.
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.run(AbstractSearchAsyncAction.java:230) ~[classes/:?]
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.executePhase(AbstractSearchAsyncAction.java:457) [classes/:?]
	... 73 more
[2022-02-22T16:25:51,780][INFO ][o.e.u.SystemIndexMigrator] [node_t0] removing read only block on [[.int-man-old/dg24NW_HS4-UwXNStbzKbw]] because reindex failed [Failed to execute phase [query], ]
[2022-02-22T16:25:51,788][INFO ][o.e.c.c.Coordinator      ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] cluster UUID [NGLyD5XQSvWnPPuueN7G8w]
[2022-02-22T16:25:51,792][WARN ][o.e.p.AllocatedPersistentTask] [node_t0] task [upgrade-system-indices] failed with an exception
org.elasticsearch.action.search.SearchPhaseExecutionException: 
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.onPhaseFailure(AbstractSearchAsyncAction.java:716) ~[classes/:?]
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.executePhase(AbstractSearchAsyncAction.java:462) ~[classes/:?]
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.start(AbstractSearchAsyncAction.java:202) ~[classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.executeSearch(TransportSearchAction.java:1031) ~[classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.executeLocalSearch(TransportSearchAction.java:747) ~[classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.lambda$executeRequest$6(TransportSearchAction.java:390) ~[classes/:?]
	at org.elasticsearch.action.ActionListener$1.onResponse(ActionListener.java:136) [classes/:?]
	at org.elasticsearch.index.query.Rewriteable.rewriteAndFetch(Rewriteable.java:112) ~[classes/:?]
	at org.elasticsearch.index.query.Rewriteable.rewriteAndFetch(Rewriteable.java:77) ~[classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.executeRequest(TransportSearchAction.java:478) ~[classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.doExecute(TransportSearchAction.java:277) ~[classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.doExecute(TransportSearchAction.java:103) ~[classes/:?]
	at org.elasticsearch.action.support.TransportAction$RequestFilterChain.proceed(TransportAction.java:79) ~[classes/:?]
	at org.elasticsearch.action.support.TransportAction.execute(TransportAction.java:54) ~[classes/:?]
	at org.elasticsearch.tasks.TaskManager.registerAndExecute(TaskManager.java:170) ~[classes/:?]
	at org.elasticsearch.client.internal.node.NodeClient.executeLocally(NodeClient.java:113) ~[classes/:?]
	at org.elasticsearch.client.internal.node.NodeClient.doExecute(NodeClient.java:91) ~[classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.execute(AbstractClient.java:380) ~[classes/:?]
	at org.elasticsearch.client.internal.FilterClient.doExecute(FilterClient.java:57) ~[classes/:?]
	at org.elasticsearch.client.internal.ParentTaskAssigningClient.doExecute(ParentTaskAssigningClient.java:55) ~[classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.execute(AbstractClient.java:380) ~[classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.search(AbstractClient.java:515) ~[classes/:?]
	at org.elasticsearch.index.reindex.ClientScrollableHitSource.doStart(ClientScrollableHitSource.java:74) ~[classes/:?]
	at org.elasticsearch.index.reindex.ScrollableHitSource.start(ScrollableHitSource.java:73) ~[classes/:?]
	at org.elasticsearch.reindex.AbstractAsyncBulkByScrollAction.start(AbstractAsyncBulkByScrollAction.java:327) ~[classes/:?]
	at org.elasticsearch.reindex.Reindexer.lambda$execute$0(Reindexer.java:119) ~[classes/:?]
	at org.elasticsearch.reindex.BulkByScrollParallelizationHelper.executeSlicedAction(BulkByScrollParallelizationHelper.java:95) ~[classes/:?]
	at org.elasticsearch.reindex.Reindexer.execute(Reindexer.java:98) ~[classes/:?]
	at org.elasticsearch.reindex.TransportReindexAction.lambda$doExecute$0(TransportReindexAction.java:103) ~[classes/:?]
	at org.elasticsearch.action.ActionListener$DelegatingFailureActionListener.onResponse(ActionListener.java:219) ~[classes/:?]
	at org.elasticsearch.reindex.BulkByScrollParallelizationHelper.initTaskState(BulkByScrollParallelizationHelper.java:125) ~[classes/:?]
	at org.elasticsearch.reindex.Reindexer.initTask(Reindexer.java:94) ~[classes/:?]
	at org.elasticsearch.reindex.TransportReindexAction.doExecute(TransportReindexAction.java:100) ~[classes/:?]
	at org.elasticsearch.reindex.TransportReindexAction.doExecute(TransportReindexAction.java:36) ~[classes/:?]
	at org.elasticsearch.action.support.TransportAction$RequestFilterChain.proceed(TransportAction.java:79) ~[classes/:?]
	at org.elasticsearch.action.support.TransportAction.execute(TransportAction.java:54) ~[classes/:?]
	at org.elasticsearch.tasks.TaskManager.registerAndExecute(TaskManager.java:170) ~[classes/:?]
	at org.elasticsearch.client.internal.node.NodeClient.executeLocally(NodeClient.java:113) ~[classes/:?]
	at org.elasticsearch.client.internal.node.NodeClient.doExecute(NodeClient.java:91) ~[classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.execute(AbstractClient.java:380) ~[classes/:?]
	at org.elasticsearch.client.internal.FilterClient.doExecute(FilterClient.java:57) ~[classes/:?]
	at org.elasticsearch.client.internal.ParentTaskAssigningClient.doExecute(ParentTaskAssigningClient.java:55) ~[classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.execute(AbstractClient.java:380) ~[classes/:?]
	at org.elasticsearch.client.internal.FilterClient.doExecute(FilterClient.java:57) ~[classes/:?]
	at org.elasticsearch.client.internal.OriginSettingClient.doExecute(OriginSettingClient.java:43) ~[classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.execute(AbstractClient.java:380) ~[classes/:?]
	at org.elasticsearch.upgrades.SystemIndexMigrator.reindex(SystemIndexMigrator.java:523) ~[classes/:?]
	at org.elasticsearch.upgrades.SystemIndexMigrator.lambda$migrateSingleIndex$14(SystemIndexMigrator.java:402) [classes/:?]
	at org.elasticsearch.action.ActionListener$1.onResponse(ActionListener.java:136) [classes/:?]
	at org.elasticsearch.action.support.ContextPreservingActionListener.onResponse(ContextPreservingActionListener.java:31) [classes/:?]
	at org.elasticsearch.cluster.AckedClusterStateUpdateTask.onAckTimeout(AckedClusterStateUpdateTask.java:71) [classes/:?]
	at org.elasticsearch.cluster.service.MasterService$ContextPreservingAckListener.onAckTimeout(MasterService.java:665) [classes/:?]
	at org.elasticsearch.cluster.service.MasterService$TaskAckListener.onTimeout(MasterService.java:765) [classes/:?]
	at org.elasticsearch.cluster.service.MasterService$TaskAckListener.onCommit(MasterService.java:719) [classes/:?]
	at org.elasticsearch.cluster.service.MasterService$CompositeTaskAckListener.onCommit(MasterService.java:778) [classes/:?]
	at org.elasticsearch.cluster.coordination.Coordinator$CoordinatorPublication$1.onCommit(Coordinator.java:1545) [classes/:?]
	at org.elasticsearch.cluster.coordination.Publication$PublicationTarget.lambda$handlePublishResponse$0(Publication.java:268) [classes/:?]
	at java.util.Optional.ifPresent(Optional.java:178) [?:?]
	at org.elasticsearch.cluster.coordination.Publication$PublicationTarget.handlePublishResponse(Publication.java:265) [classes/:?]
	at org.elasticsearch.cluster.coordination.Publication$PublicationTarget$PublishResponseHandler.onResponse(Publication.java:364) [classes/:?]
	at org.elasticsearch.cluster.coordination.Publication$PublicationTarget$PublishResponseHandler.onResponse(Publication.java:341) [classes/:?]
	at org.elasticsearch.cluster.coordination.Coordinator$4.onResponse(Coordinator.java:1366) [classes/:?]
	at org.elasticsearch.cluster.coordination.PublicationTransportHandler$PublicationContext$1.onResponse(PublicationTransportHandler.java:354) [classes/:?]
	at org.elasticsearch.cluster.coordination.PublicationTransportHandler$PublicationContext$1.onResponse(PublicationTransportHandler.java:350) [classes/:?]
	at org.elasticsearch.action.ActionListener$DelegatingActionListener.onResponse(ActionListener.java:186) [classes/:?]
	at org.elasticsearch.action.ActionListener$RunAfterActionListener.onResponse(ActionListener.java:341) [classes/:?]
	at org.elasticsearch.action.ActionListener$RunAfterActionListener.onResponse(ActionListener.java:341) [classes/:?]
	at org.elasticsearch.action.ActionListenerResponseHandler.handleResponse(ActionListenerResponseHandler.java:43) [classes/:?]
	at org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleResponse(TransportService.java:1339) [classes/:?]
	at org.elasticsearch.transport.TransportService$DirectResponseChannel.processResponse(TransportService.java:1417) [classes/:?]
	at org.elasticsearch.transport.TransportService$DirectResponseChannel$1.run(TransportService.java:1402) [classes/:?]
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:717) [classes/:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: org.elasticsearch.action.search.SearchPhaseExecutionException: Search rejected due to missing shards [[.int-man-old][0]]. Consider using `allow_partial_search_results` setting to bypass this error.
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.run(AbstractSearchAsyncAction.java:230) ~[classes/:?]
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.executePhase(AbstractSearchAsyncAction.java:457) ~[classes/:?]
	... 73 more
[2022-02-22T16:25:51,794][WARN ][o.e.p.PersistentTasksClusterService] [node_t0] persistent task upgrade-system-indices failed
org.elasticsearch.action.search.SearchPhaseExecutionException: 
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.onPhaseFailure(AbstractSearchAsyncAction.java:716) ~[classes/:?]
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.executePhase(AbstractSearchAsyncAction.java:462) ~[classes/:?]
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.start(AbstractSearchAsyncAction.java:202) ~[classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.executeSearch(TransportSearchAction.java:1031) ~[classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.executeLocalSearch(TransportSearchAction.java:747) ~[classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.lambda$executeRequest$6(TransportSearchAction.java:390) ~[classes/:?]
	at org.elasticsearch.action.ActionListener$1.onResponse(ActionListener.java:136) ~[classes/:?]
	at org.elasticsearch.index.query.Rewriteable.rewriteAndFetch(Rewriteable.java:112) ~[classes/:?]
	at org.elasticsearch.index.query.Rewriteable.rewriteAndFetch(Rewriteable.java:77) ~[classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.executeRequest(TransportSearchAction.java:478) ~[classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.doExecute(TransportSearchAction.java:277) ~[classes/:?]
	at org.elasticsearch.action.search.TransportSearchAction.doExecute(TransportSearchAction.java:103) ~[classes/:?]
	at org.elasticsearch.action.support.TransportAction$RequestFilterChain.proceed(TransportAction.java:79) ~[classes/:?]
	at org.elasticsearch.action.support.TransportAction.execute(TransportAction.java:54) ~[classes/:?]
	at org.elasticsearch.tasks.TaskManager.registerAndExecute(TaskManager.java:170) ~[classes/:?]
	at org.elasticsearch.client.internal.node.NodeClient.executeLocally(NodeClient.java:113) ~[classes/:?]
	at org.elasticsearch.client.internal.node.NodeClient.doExecute(NodeClient.java:91) ~[classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.execute(AbstractClient.java:380) ~[classes/:?]
	at org.elasticsearch.client.internal.FilterClient.doExecute(FilterClient.java:57) ~[classes/:?]
	at org.elasticsearch.client.internal.ParentTaskAssigningClient.doExecute(ParentTaskAssigningClient.java:55) ~[classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.execute(AbstractClient.java:380) ~[classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.search(AbstractClient.java:515) ~[classes/:?]
	at org.elasticsearch.index.reindex.ClientScrollableHitSource.doStart(ClientScrollableHitSource.java:74) ~[classes/:?]
	at org.elasticsearch.index.reindex.ScrollableHitSource.start(ScrollableHitSource.java:73) ~[classes/:?]
	at org.elasticsearch.reindex.AbstractAsyncBulkByScrollAction.start(AbstractAsyncBulkByScrollAction.java:327) ~[classes/:?]
	at org.elasticsearch.reindex.Reindexer.lambda$execute$0(Reindexer.java:119) ~[classes/:?]
	at org.elasticsearch.reindex.BulkByScrollParallelizationHelper.executeSlicedAction(BulkByScrollParallelizationHelper.java:95) ~[classes/:?]
	at org.elasticsearch.reindex.Reindexer.execute(Reindexer.java:98) ~[classes/:?]
	at org.elasticsearch.reindex.TransportReindexAction.lambda$doExecute$0(TransportReindexAction.java:103) ~[classes/:?]
	at org.elasticsearch.action.ActionListener$DelegatingFailureActionListener.onResponse(ActionListener.java:219) ~[classes/:?]
	at org.elasticsearch.reindex.BulkByScrollParallelizationHelper.initTaskState(BulkByScrollParallelizationHelper.java:125) ~[classes/:?]
	at org.elasticsearch.reindex.Reindexer.initTask(Reindexer.java:94) ~[classes/:?]
	at org.elasticsearch.reindex.TransportReindexAction.doExecute(TransportReindexAction.java:100) ~[classes/:?]
	at org.elasticsearch.reindex.TransportReindexAction.doExecute(TransportReindexAction.java:36) ~[classes/:?]
	at org.elasticsearch.action.support.TransportAction$RequestFilterChain.proceed(TransportAction.java:79) ~[classes/:?]
	at org.elasticsearch.action.support.TransportAction.execute(TransportAction.java:54) ~[classes/:?]
	at org.elasticsearch.tasks.TaskManager.registerAndExecute(TaskManager.java:170) ~[classes/:?]
	at org.elasticsearch.client.internal.node.NodeClient.executeLocally(NodeClient.java:113) ~[classes/:?]
	at org.elasticsearch.client.internal.node.NodeClient.doExecute(NodeClient.java:91) ~[classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.execute(AbstractClient.java:380) ~[classes/:?]
	at org.elasticsearch.client.internal.FilterClient.doExecute(FilterClient.java:57) ~[classes/:?]
	at org.elasticsearch.client.internal.ParentTaskAssigningClient.doExecute(ParentTaskAssigningClient.java:55) ~[classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.execute(AbstractClient.java:380) ~[classes/:?]
	at org.elasticsearch.client.internal.FilterClient.doExecute(FilterClient.java:57) ~[classes/:?]
	at org.elasticsearch.client.internal.OriginSettingClient.doExecute(OriginSettingClient.java:43) ~[classes/:?]
	at org.elasticsearch.client.internal.support.AbstractClient.execute(AbstractClient.java:380) ~[classes/:?]
	at org.elasticsearch.upgrades.SystemIndexMigrator.reindex(SystemIndexMigrator.java:523) ~[classes/:?]
	at org.elasticsearch.upgrades.SystemIndexMigrator.lambda$migrateSingleIndex$14(SystemIndexMigrator.java:402) ~[classes/:?]
	at org.elasticsearch.action.ActionListener$1.onResponse(ActionListener.java:136) ~[classes/:?]
	at org.elasticsearch.action.support.ContextPreservingActionListener.onResponse(ContextPreservingActionListener.java:31) ~[classes/:?]
	at org.elasticsearch.cluster.AckedClusterStateUpdateTask.onAckTimeout(AckedClusterStateUpdateTask.java:71) ~[classes/:?]
	at org.elasticsearch.cluster.service.MasterService$ContextPreservingAckListener.onAckTimeout(MasterService.java:665) ~[classes/:?]
	at org.elasticsearch.cluster.service.MasterService$TaskAckListener.onTimeout(MasterService.java:765) ~[classes/:?]
	at org.elasticsearch.cluster.service.MasterService$TaskAckListener.onCommit(MasterService.java:719) ~[classes/:?]
	at org.elasticsearch.cluster.service.MasterService$CompositeTaskAckListener.onCommit(MasterService.java:778) ~[classes/:?]
	at org.elasticsearch.cluster.coordination.Coordinator$CoordinatorPublication$1.onCommit(Coordinator.java:1545) ~[classes/:?]
	at org.elasticsearch.cluster.coordination.Publication$PublicationTarget.lambda$handlePublishResponse$0(Publication.java:268) ~[classes/:?]
	at java.util.Optional.ifPresent(Optional.java:178) ~[?:?]
	at org.elasticsearch.cluster.coordination.Publication$PublicationTarget.handlePublishResponse(Publication.java:265) ~[classes/:?]
	at org.elasticsearch.cluster.coordination.Publication$PublicationTarget$PublishResponseHandler.onResponse(Publication.java:364) ~[classes/:?]
	at org.elasticsearch.cluster.coordination.Publication$PublicationTarget$PublishResponseHandler.onResponse(Publication.java:341) ~[classes/:?]
	at org.elasticsearch.cluster.coordination.Coordinator$4.onResponse(Coordinator.java:1366) ~[classes/:?]
	at org.elasticsearch.cluster.coordination.PublicationTransportHandler$PublicationContext$1.onResponse(PublicationTransportHandler.java:354) ~[classes/:?]
	at org.elasticsearch.cluster.coordination.PublicationTransportHandler$PublicationContext$1.onResponse(PublicationTransportHandler.java:350) ~[classes/:?]
	at org.elasticsearch.action.ActionListener$DelegatingActionListener.onResponse(ActionListener.java:186) ~[classes/:?]
	at org.elasticsearch.action.ActionListener$RunAfterActionListener.onResponse(ActionListener.java:341) ~[classes/:?]
	at org.elasticsearch.action.ActionListener$RunAfterActionListener.onResponse(ActionListener.java:341) ~[classes/:?]
	at org.elasticsearch.action.ActionListenerResponseHandler.handleResponse(ActionListenerResponseHandler.java:43) ~[classes/:?]
	at org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleResponse(TransportService.java:1339) ~[classes/:?]
	at org.elasticsearch.transport.TransportService$DirectResponseChannel.processResponse(TransportService.java:1417) ~[classes/:?]
	at org.elasticsearch.transport.TransportService$DirectResponseChannel$1.run(TransportService.java:1402) ~[classes/:?]
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:717) ~[classes/:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: org.elasticsearch.action.search.SearchPhaseExecutionException: Search rejected due to missing shards [[.int-man-old][0]]. Consider using `allow_partial_search_results` setting to bypass this error.
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.run(AbstractSearchAsyncAction.java:230) ~[classes/:?]
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.executePhase(AbstractSearchAsyncAction.java:457) ~[classes/:?]
	... 73 more
[2022-02-22T16:25:51,803][WARN ][o.e.p.AllocatedPersistentTask] [node_t0] attempt to complete task [upgrade-system-indices[c]] with id [upgrade-system-indices] in the [COMPLETED] state
[2022-02-22T16:25:51,809][INFO ][o.e.c.s.MasterService    ] [node_t0] node-join[{node_t1}{ntz6Qb7ZQvusv1DZfmoRkg}{921plgKxSjq7bij6y1k_ew}{127.0.0.1}{127.0.0.1:59546}{cdfhilmrstw} joining after restart, removed [0ms] ago with reason [disconnected]], term: 1, version: 14, delta: added {{node_t1}{ntz6Qb7ZQvusv1DZfmoRkg}{921plgKxSjq7bij6y1k_ew}{127.0.0.1}{127.0.0.1:59546}{cdfhilmrstw}}
[2022-02-22T16:25:51,818][INFO ][o.e.c.s.ClusterApplierService] [node_t1] master node changed {previous [], current [{node_t0}{vvb_ksGqTQ6iTJrASggbnw}{oS7cOM2YR4WhH0rvPZxNnw}{127.0.0.1}{127.0.0.1:59521}{m}]}, added {{node_t0}{vvb_ksGqTQ6iTJrASggbnw}{oS7cOM2YR4WhH0rvPZxNnw}{127.0.0.1}{127.0.0.1:59521}{m}}, term: 1, version: 14, reason: ApplyCommitRequest{term=1, version=14, sourceNode={node_t0}{vvb_ksGqTQ6iTJrASggbnw}{oS7cOM2YR4WhH0rvPZxNnw}{127.0.0.1}{127.0.0.1:59521}{m}}
[2022-02-22T16:25:51,818][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] started
[2022-02-22T16:25:51,819][INFO ][o.e.c.s.ClusterApplierService] [node_t0] added {{node_t1}{ntz6Qb7ZQvusv1DZfmoRkg}{921plgKxSjq7bij6y1k_ew}{127.0.0.1}{127.0.0.1:59546}{cdfhilmrstw}}, term: 1, version: 14, reason: Publication{term=1, version=14}
[2022-02-22T16:25:51,822][INFO ][o.e.m.SystemIndexMigrationIT] [testSystemIndexMigrationCanBeInterruptedWithShutdown] [SystemIndexMigrationIT#testSystemIndexMigrationCanBeInterruptedWithShutdown]: cleaning up after test
[2022-02-22T16:25:51,874][INFO ][o.e.c.m.MetadataDeleteIndexService] [node_t0] [.int-man-old/dg24NW_HS4-UwXNStbzKbw] deleting index
[2022-02-22T16:25:51,874][INFO ][o.e.c.m.MetadataDeleteIndexService] [node_t0] [.int-man-old-reindexed-for-8/4B7GFl-IROyDfrsSnAFETA] deleting index
[2022-02-22T16:25:51,898][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] stopping ...
[2022-02-22T16:25:51,900][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] stopped
[2022-02-22T16:25:51,900][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] closing ...
[2022-02-22T16:25:51,900][INFO ][o.e.t.ClusterConnectionManager] [node_t1] transport connection to [{node_t0}{vvb_ksGqTQ6iTJrASggbnw}{oS7cOM2YR4WhH0rvPZxNnw}{127.0.0.1}{127.0.0.1:59521}{m}] closed by remote
[2022-02-22T16:25:51,900][INFO ][o.e.c.c.Coordinator      ] [node_t1] master node [{node_t0}{vvb_ksGqTQ6iTJrASggbnw}{oS7cOM2YR4WhH0rvPZxNnw}{127.0.0.1}{127.0.0.1:59521}{m}] disconnected, restarting discovery
[2022-02-22T16:25:51,901][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] closed
[2022-02-22T16:25:51,901][INFO ][o.e.c.s.ClusterApplierService] [node_t1] master node changed {previous [{node_t0}{vvb_ksGqTQ6iTJrASggbnw}{oS7cOM2YR4WhH0rvPZxNnw}{127.0.0.1}{127.0.0.1:59521}{m}], current []}, term: 1, version: 18, reason: becoming candidate: onLeaderFailure
[2022-02-22T16:25:51,901][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] stopping ...
[2022-02-22T16:25:51,903][WARN ][o.e.c.NodeConnectionsService] [node_t1] failed to connect to {node_t0}{vvb_ksGqTQ6iTJrASggbnw}{oS7cOM2YR4WhH0rvPZxNnw}{127.0.0.1}{127.0.0.1:59521}{m} (tried [1] times)
org.elasticsearch.transport.ConnectTransportException: [node_t0][127.0.0.1:59521] connect_exception
	at org.elasticsearch.transport.TcpTransport$ChannelsConnectedListener.onFailure(TcpTransport.java:1107) ~[classes/:?]
	at org.elasticsearch.action.ActionListener.lambda$toBiConsumer$0(ActionListener.java:279) ~[classes/:?]
	at org.elasticsearch.core.CompletableContext.lambda$addListener$0(CompletableContext.java:31) ~[classes/:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2162) ~[?:?]
	at org.elasticsearch.core.CompletableContext.completeExceptionally(CompletableContext.java:46) ~[classes/:?]
	at org.elasticsearch.transport.netty4.Netty4TcpChannel.lambda$addListener$0(Netty4TcpChannel.java:63) ~[classes/:?]
	at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578) ~[netty-common-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.util.concurrent.DefaultPromise.notifyListeners0(DefaultPromise.java:571) ~[netty-common-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:550) ~[netty-common-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491) ~[netty-common-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616) ~[netty-common-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.util.concurrent.DefaultPromise.setFailure0(DefaultPromise.java:609) ~[netty-common-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.util.concurrent.DefaultPromise.tryFailure(DefaultPromise.java:117) ~[netty-common-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.fulfillConnectPromise(AbstractNioChannel.java:321) ~[netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:337) ~[netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:710) ~[netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysPlain(NioEventLoop.java:623) ~[netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:586) ~[netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496) ~[netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986) ~[netty-common-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.73.Final.jar:4.1.73.Final]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: 127.0.0.1/127.0.0.1:59521
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.Net.pollConnect(Native Method) ~[?:?]
	at sun.nio.ch.Net.pollConnectNow(Net.java:672) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:946) ~[?:?]
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:330) ~[netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:334) ~[netty-transport-4.1.73.Final.jar:4.1.73.Final]
	... 7 more
[2022-02-22T16:25:51,904][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] stopped
[2022-02-22T16:25:51,904][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] closing ...
[2022-02-22T16:25:51,905][INFO ][o.e.n.Node               ] [testSystemIndexMigrationCanBeInterruptedWithShutdown] closed
[2022-02-22T16:25:51,905][INFO ][o.e.m.SystemIndexMigrationIT] [testSystemIndexMigrationCanBeInterruptedWithShutdown] [SystemIndexMigrationIT#testSystemIndexMigrationCanBeInterruptedWithShutdown]: cleaned up after test
[2022-02-22T16:25:51,908][INFO ][o.e.m.SystemIndexMigrationIT] [testSystemIndexMigrationCanBeInterruptedWithShutdown] after test
Feb 22, 2022 11:40:51 AM com.carrotsearch.randomizedtesting.ThreadLeakControl checkThreadLeaks
WARNING: Will linger awaiting termination of 1 leaked thread(s).

Process finished with exit code 0

```

Reply:
I looked into this and it appears to be because we restart the only data node, so the reindex fails. This makes the migration fail, but that's okay for this test - we don't care that the migration fails, just that the node can come back online after we restart it while the migration is in progress.